### PR TITLE
docs: Simulate HTTP faults

### PR DESCRIPTION
**Pull request by en-jin19**
Signed-off-by: en-jin19 <jinenwei@pingcap.com>

Translated from: https://github.com/chaos-mesh/website/pull/168

### CODE REVIEW

**Comment by en-jin19** [comment]
@Hexilee Could you please provide the English image of "create-new-httpchaos.png"? thanks!

**Comment by Hexilee** [comment]
> @Hexilee Could you please provide the English image of "create-new-httpchaos.png"? thanks!
create-http-chaos

Is this picture ok?

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -26,9 +26,23 @@ Before injecting the faults supported by HTTPChaos, note the followings:
...
+
+![create HTTP fault](./img/create-new-httpchaos.png)
+
+3. Submit the experiment.
+
+Take the above image as an example, click the "**submit**" button means that you have interrupted the configurations to the responses of all requests of Port 80.
+
+## Create experiments using YAML files
+
+Chaos Mesh also supports using YAML configuration files to create HTTPChaos experiments. In the YAML files, you can simulate either one HTTP fault type or a combination of different HTTP fault types.
```
Comment (ran-huang) [inline docs/simulate-http-chaos-on-kubernetes.md:45]:
```suggestion
Chaos Mesh also supports using YAML configuration files to create HTTPChaos experiments. In a YAML file, you can simulate either one HTTP fault type or a combination of different HTTP fault types.
```

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 2]
```diff
@@ -112,22 +122,22 @@ Common fields are meaningful when the `target` of fault injection is `Request` o
...
-| scheduler | string | Specifies the scheduling rules for the time of a specific experiment. |  | no | 5 \* \* \* \* |
+| `mode` | string | Specifies the mode of the experiment. The mode options include `one` (selecting a random pod), `all` (selecting all eligible pods), `fixed` (selecting a specified number of eligible pods), `fixed-percent` (selecting a specified percentage of Pods from the eligible pods), and `random-max-percent` (selecting the maximum percentage of Pods from the eligible pods). |  | yes | `one` |
+| `value` | string | Provides parameters for the `mode` configuration depending on the value of `mode`. |  | no | 1 |
+| `target` | string | Specifies whether the target of fault injuection is `Request` or `Response`. The [`target`-related fields](#Description-for-`target`-related-fields) should be configured at the same time. |  | yes | Request |
+| `port` | int32 | The TCP port that the target service listens on. |  | yes | 80 |
+| `path` | string | The URI path of the target request which supports [Matching wildcards](https://www.wikiwand.com/en/Matching_wildcards). | Takes effect on all paths by default. | no | /api/\* |
```
Comment (ran-huang) [inline docs/simulate-http-chaos-on-kubernetes.md:129]:
```suggestion
| `path` | string | The URI path of the target request. Supports [Matching wildcards](https://www.wikiwand.com/en/Matching_wildcards). | Takes effect on all paths by default. | no | /api/\* |
```
支持通配符的是 path，不是 request。

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 3]
```diff
@@ -26,9 +26,23 @@ Before injecting the faults supported by HTTPChaos, note the followings:
 - For HTTPChaos injection to take effect, the client should avoid reusing TCP socket. This is because HTTPChaos does not affect the HTTP requests that are sent via TCP socket before the fault injection.
 - Use non-idempotent requests (such as most of the POST requests) with caution in production environments. If such requests are used, the target service may not return to normal status by repeating requests after the fault injection.
 
-## Create experiments
+## Create experiments using Chaos Dashboard
 
-Currently, Chaos Mesh only supports using YAML configuration files to create HTTPChaos experiments. In the YAML files, you can simulate either one HTTP fault type or a combination of different HTTP fault types.
+1. Open Chaos Dashboard, and click "**NEW EXPERIMENT**" on the page to create a new experiment:
```
[the remark this answers is not in the document]
Comment (en-jin19) [reply docs/simulate-http-chaos-on-kubernetes.md:31]: but in our Chaos dashboard and the attached image, these words are all capitalized. How about we make the text consistent with the corresponding ones in images?

**Review by Hexilee — APPROVED** [review]
LGTM. How about changing the title to "docs-en: add dashboard usage guide for HTTPChaos", I think the current title does not suit as a contents updating.

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ran-huang) [reply docs/simulate-http-chaos-on-kubernetes.md:31]: LGTM

**Comment by en-jin19** [comment]
@ran-huang I have accepted the comments, PTAL, thanks!

**Comment by en-jin19** [comment]
@ran-huang all comments have been fixed, PTAL, thanks!

**Review by ran-huang — APPROVED** [review]

**Pull request merged**